src/components/AskQuestion/AskQuestion.vue

```diff
@@ -117,13 +117,17 @@ export default {
       // initiated with send-button. questionToList will be new stringify-Entry and will be pushed in array - later new DB-entry.
       // todo: check min-length of title/description?
       // afterwards delete this.title, this.description. Later on have to check all the attributes.
-      this.questionCreated_at = new Date();
+      let fullDate = new Date();
+      let month = fullDate.getMonth() + 1 + ".";
+      let day = fullDate.getDate() + ".";
+      let year = fullDate.getFullYear();
+      this.questionCreated_at = day + month + year;
```

Comment:
Please use a Template String for this: https://developer.mozilla.org/de/docs/Web/JavaScript/Reference/Template_literals

Reply:
need more information? 

why? what?